src/basilisp/core.lpy

```diff
@@ -1094,14 +1094,16 @@
   (operator/mod num div))
 
 (defn ^:inline quot
-  "Returns the quotient of ``num`` and ``div``\\."
+  "Returns the quotient of ``num`` and ``div``\\.
+
+  The result is rounded towards negative infinity."
   [num div]
   (basilisp.lang.runtime/quotient num div))
 
 (defn ^:inline rem
```

Comment:
```suggestion
(defn rem
```

This can no longer be inlined due to the dependency on `math`.

Reply:
Done,

I'm wondering whether it is possible (and worth the effort) to develop an automated test that will load all basilisp libs and try to call all :inline fn in an bare namespace so as to catch these violations?

Reply:
If you want to file a ticket and explore it, please do. My guess is that it would just be more fruitful to move the core test suite entirely over to Basilisp code (e.g. remove `core_test.py`) since those functions would be compiled in as inlined code.

Trying to dynamically call every function would require that we had arguments for those functions, which is basically just what tests are.